src/components/tooltip/Marker.js

```diff
@@ -208,496 +62,6 @@ function _maybeFormatDuration(
-                  {_makePhaseTimesArray(nursery.phase_times)
-                    /*
-                     * Nursery collection should usually be very quick.  1ms
-                     * is good and beyond 5ms and we could cause some
-                     * animation to drop frames.  250us is about where
-                     * things start to get interesting for a phase of a
-                     * nursery collection.
-                     */
-                    .filter(pt => pt.time > 250) // 250us
-                    .map(_markerDetailPhase)}
-                </TooltipDetails>
-              );
-              break;
-            }
-            case 'nursery disabled':
-              tooltipDetails = (
-                <TooltipDetails>
-                  <TooltipDetail label="Status">Nursery disabled</TooltipDetail>
-                </TooltipDetails>
-              );
-              break;
-            case 'nursery empty':
-              tooltipDetails = (
-                <TooltipDetails>
-                  <TooltipDetail label="Status">Nursery empty</TooltipDetail>
-                </TooltipDetails>
-              );
-              break;
-            default:
-              return null;
-          }
-        } else {
-          return null;
-        }
-        break;
-      }
-      case 'GCMajor': {
-        const timings = data.timings;
-        switch (timings.status) {
-          case 'aborted':
-            tooltipDetails = (
-              <TooltipDetails>
-                <TooltipDetail label="Status">Aborted (OOM)</TooltipDetail>
-              </TooltipDetails>
-            );
-            break;
-          case 'completed': {
-            let nonIncrementalReason = null;
-            if (
-              timings.nonincremental_reason &&
-              timings.nonincremental_reason !== 'None'
-            ) {
-              nonIncrementalReason = (
-                <TooltipDetail label="Non-incremental reason">
-                  {timings.nonincremental_reason}
-                </TooltipDetail>
-              );
-            }
-            const phase_times = _filterInterestingPhaseTimes(
-              timings.phase_times,
-              6
-            );
-            let gcsize;
-            const post_heap_size = timings.post_heap_size;
-            if (post_heap_size !== undefined) {
-              gcsize = (
-                <TooltipDetail label="Heap size (pre - post)">
-                  {formatBytes(timings.allocated_bytes) +
-                    ' - ' +
-                    formatBytes(post_heap_size)}
-                </TooltipDetail>
-              );
-            } else {
-              gcsize = (
-                <TooltipDetail label="Heap size (pre)">
-                  {formatBytes(timings.allocated_bytes)}
-                </TooltipDetail>
-              );
-            }
-            tooltipDetails = (
-              <TooltipDetails>
-                <TooltipDetail label="Reason">{timings.reason}</TooltipDetail>
```

Comment:
We also don't have reason in the new code. I started to feel like I'm missing something here 😄

Reply:
I have no idea how this happened.